whylog/tests/tests_client/test_backtrackerSearcher.py

```diff
@@ -36,12 +36,20 @@ def _verify_lines(self, lines_normally, lines_reversed):
             assert i == j
 
     def _run_reverse_and_check_results(
-        self, log_file_path, reverse_from_offset_params, how_many_last_lines
+        self,
+        log_file_path,
+        reverse_from_offset_params,
+        how_many_last_lines,
+        final_offset_should_be_zero=True
```

Comment:
I've tried, but yapf (command line: `yapf -ir .` in main repo directory) has reverted my changes :( So I can't aplly this proposal.